src/anemoi/training/data/datamodule.py

```diff
@@ -96,6 +97,9 @@ def __init__(self, config: DictConfig) -> None:
             )
             self.config.dataloader.training.end = self.config.dataloader.validation.start - 1
 
+        if not self.config.dataloader.pin_memory:
```

Comment:
Please adjust this config retrieval to use a `.get` call with a default value to reduce config issues.
```python
if not self.config.dataloader.get('pin_memory', True):
```

Reply:
Thanks for your quick review. I've implemented you change requests.